libraries/k8sversion.rb

```diff
@@ -14,7 +15,7 @@ class K8sVersion < K8sResourceBase
         ...
       end
     "
-    attr_reader :k8sversion, :k8suri, :gitVersion, :major, :minor, :platform, :buildDate
+    attr_reader :k8sversion, :k8suri, :gitVersion, :major, :minor, :buildDate
```

Comment:
We have a method platform in the same class, so lint gave a duplicate method error.